Dockerfile

```diff
@@ -56,6 +56,16 @@ RUN apt-get update \
   wget \
 && apt-get -y autoremove && apt-get -y autoclean && apt-get clean && rm -rf /var/lib/apt/lists /tmp/* /var/tmp/*
 
+# NewRelic is disabled by default.
+ENV NEW_RELIC_ENABLED=false
+
+# Install NewRelic agent https://docs.newrelic.com/docs/agents/php-agent/installation/php-agent-installation-ubuntu-debian
+RUN echo 'deb http://apt.newrelic.com/debian/ newrelic non-free' | tee /etc/apt/sources.list.d/newrelic.list && \
+    wget -O- https://download.newrelic.com/548C16BF.gpg | apt-key add - && \
+    apt-get update && \
+    apt-get install -y newrelic-php5 && \
+    rm -f /etc/php/7.2/mods-available/newrelic.ini
```

Comment:
That will leave empty, dangling symlinks in:
/etc/php/7.2/apache2/conf.d/20-newrelic.ini & 
/etc/php/7.2/cli/conf.d/20-newrelic.ini
I think

Reply:
Fixed!